org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -34,8 +35,16 @@
 	private DotnetNewWizardPage wizardPage;
 	@Override
 	public void init(IWorkbench workbench, IStructuredSelection selection) {
-		// TODO
 		wizardPage = new DotnetNewWizardPage();
+
+		Object firstElement = selection.getFirstElement();
+		if (firstElement instanceof IResource) {
```

Comment:
You're right, sorry. I think you have to iterate on all working sets with `PlatformUI.getWorkbench().getWorkingSetManager().getAllWorkingSets()` to check whether the `getElements()` contains one of the selected items.
The `ResourceWokringSetPage` is probably worth having a look as a reference.